saleor/graphql/core/tests/test_core.py

```diff
@@ -319,6 +322,111 @@ def test_validate_image_file_invalid_file_extension():
     )
 
 
+def test_get_filename_from_url_unique():
+    # given
+    file_format = "jpg"
+    file_name = "lenna"
+    url = f"http://example.com/{file_name}.{file_format}"
+
+    # when
+    result = get_filename_from_url(url)
+
+    # then
+    assert result.startswith(file_name)
+    assert result.endswith(file_format)
+    assert result != f"{file_name}.{file_format}"
+
+
+def test_is_image_mimetype_valid_mimetype():
+    # given
+    valid_mimetype = "image/jpeg"
+
+    # when
+    result = is_image_mimetype(valid_mimetype)
+
+    # then
+    assert result
```

Comment:
Could be just `assert is_image_mimetype("image/jpeg")`

Reply:
Talked with Marcin - wanted to keep consistency with the other tests and it can stay